src/components/navBar/navBar.js

```diff
@@ -414,6 +418,7 @@ function MdNavItem($$rAF) {
       'mdNavHref': '@?',
       'mdNavSref': '@?',
       'name': '@',
+      'disabled': '=?ngDisabled',
```

Comment:
I'm using the ngDisabled attribute here because of the tabs directive which is also using ngDisabled.
I feel like the behaviour of those components should be as equal as possible.
So I think when we use the disabled attribute here we should change it in the tabs component too.